dataprofiler/data_readers/data_utils.py

```diff
@@ -553,12 +555,21 @@ def load_as_str_from_file(file_path, file_encoding, max_lines=10,
             
             # Return either the last index of sample_lines OR
             # the index of the newline char that matches remaining_lines
+            search_query_value = '\n'
+            if (isinstance(sample_lines, bytes)):
+                search_query_value = b'\r\n'
+            
             loc, occurance = find_nth_loc(sample_lines,
-                                          search_query='\n',
-                                          n=remaining_lines)
+                                        search_query=search_query_value,
```

Comment:
these should be in line with the function I believe

Reply:
@gautomdas you can use the previous version in the diff for reference, `search_query` should be in line with `sample_lines` and same for `n=...`